server/src/com/thoughtworks/go/server/service/UserService.java

```diff
@@ -203,6 +203,10 @@ public void modifyRolesAndUserAdminPrivileges(final List<String> users, final Tr
         return roles;
     }
 
+    public Collection<String> allRoleNames() {
+        return allRoleNames(goConfigService.cruiseConfig());
+    }
+
     public Collection<Role> allRoles(CruiseConfig cruiseConfig) {
```

Comment:
This method is only used in specs and the scope change it introduces is used only is specs is there a way to avoid that?